lodash.src.js

```diff
@@ -138,6 +138,8 @@
     return RegExp(upper + '+(?=' + upper + lower + ')|' + upper + '?' + lower + '|' + upper + '+|[0-9]+', 'g');
   }());
 
+  var reLines = /[^\r?\n|\u2028|\u2029|\u000a|\u000b|\u000c]+/g;
```

Comment:
`\u000a` is `\r`
`\u000b` is `\v`
`\u000c` is `\f`

Line terminators are specified [here](http://www.ecma-international.org/ecma-262/6.0/index.html#sec-line-terminators):